### ISSUE TITLE
Set custom icon markers

### ISSUE THREAD

**Issue by felipdcz** [bernardodiasc/x-world-map#15]
As a designer,
I want to be able to set different icons for the markers for each type and category,
So that I will have custom icons for Profiles, Events, Places, Locals, and respective categories
And I need to be able to change them later in a CMS so it can be iterated over.

Concept example:
![image](https://user-images.githubusercontent.com/3913116/182398317-6eba7da8-e739-4ef3-b791-c1e95b44b872.png)

![image](https://user-images.githubusercontent.com/3913116/182397115-05cf969f-e5be-487d-9b90-06adfbfd48f6.png)

**Comment by felipdcz** [comment]
There is a card for this task already.
https://github.com/bernardodiasc/x-world-map/projects/7#card-79149245

**Comment by felipdcz** [comment]
Got it, I will continue here if it's ok.

Can we have one icon marker for the option 'multiple categories'?

Example: 
- if it's a Cowork event, show a 'coworking' icon,
- if it's a Cowork event and a Conference, show the 'multiple categories' icon.

**Comment by bernardodiasc** [comment]
Yeah, we can choose any icon for the markers. That's not a problem. The thing that we need to take care first on this scenario is how multiple content-types will be displayed once you click in the marker.

In relation to the Issue vs. Card, in this case I'd suggest to delete the card (copy any useful note) and leave only this issue. Also try to add this issue (and the others you created) to the Projects Kanban board.

Do you have access to use this?

![screenshot-github com-2022 08 04-10_17_49](https://user-images.githubusercontent.com/131859/182857314-23912662-0212-4531-a612-b6d6249cce34.png)

**Comment by bernardodiasc** [comment]
By the way, we have a library of icons that you can see here https://x-world-map.com/docs?path=/story/generic-svg-use-cases--prop-name-marker-svg

![2022-08-04 10 25 23](https://user-images.githubusercontent.com/131859/182858499-f10aa8c8-776a-4756-a963-48ce82a2f2eb.gif)


Also, the icon set I've been using is the https://icons8.com/icons/stickers.
